packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -106,13 +105,15 @@ const FeedbackStore = types
     }
 
     function update (annotation) {
-      const { task, value } = annotation
-      const taskRules = self.rules.get(task) || []
-      const updatedTaskRules = taskRules.map(rule => {
-        const ruleReducer = strategies[rule.strategy].reducer
-        return ruleReducer(rule, value)
-      })
-      self.rules.set(task, updatedTaskRules)
+      if (self.isActive) {
+        const { task, value } = annotation
+        const taskRules = self.rules.get(task) || []
```

Comment:
Maybe the task rules should default to `undefined` instead of `[]`? I'm fairly certain that trying to set an undefined key on an object doesn't create a new key.